features/steps/naemon.py

```diff
@@ -0,0 +1,89 @@
+from behave import given, when, then, use_step_matcher
+import subprocess
+
+use_step_matcher('re')
+
+
+@given('I have naemon (?P<object_type>.+) objects')
+def add_naemon_object(context, object_type):
+    print ("Object type %s" % object_type)
+    for row in context.table:
+        context.naemonobjconfig.add_obj(object_type, context.table)
+
+
+@given('I have naemon config (?P<parameter>.+) set to (?P<value>.+)')
+def set_naemon_parameter(context, parameter, value):
+    context.naemonsysconfig.set_var(parameter, value)
+
+
+@given('I have an invalid naemon system configuration')
+def invalid_sysconfig(context):
+    context.execute_steps(u'Given I have naemon config invalid_param set to x')
+
+
+@given('I have an invalid naemon object configuration')
+def invalid_objconfig(context):
+    context.execute_steps(u'''
+        Given I have naemon host objects
+            | use          | host_name    | address   | check_command |
+            | default-host | invalid_host | 127.0.0.1 | non_existing  |
+    ''')
+
+
+@given('I write config to file')
+def configuration_to_file(context):
+    context.naemonobjconfig.to_file(context.naemonobjconfig.filename)
+    context.naemonsysconfig.to_file(context.naemonsysconfig.filename)
+    context.execute_steps(u'Given I have directory checkresults')
+
+
+@given('I verify the naemon configuration')
+def config_verification(context):
+    context.execute_steps(u'Given I write config to file')
+    executable = (
+        context.wrkdir + '/' + context.config.userdata['naemon_exec_path']
+    )
+    args = [executable, '--allow-root', '-v', context.naemonsysconfig.filename]
```

Comment:
Not without Flake8 complaining, the width of the row would exceed 79 and we'd have to introduce line breaks or make it into a variable with a shorter name. Personally I don't think it's worth it, do you?